plugins/aws/resoto_plugin_aws/resource/elasticbeanstalk.py

```diff
@@ -103,4 +122,214 @@ def delete_resource(self, client: AwsClient) -> bool:
+        "environment_links": S("EnvironmentLinks", default=[]) >> ForallBend(AwsBeanstalkEnvironmentLink.mapping),
+        "operations_role": S("OperationsRole"),
+    }
+    application_name: Optional[str] = field(default=None)
+    version_label: Optional[str] = field(default=None)
+    solution_stack_name: Optional[str] = field(default=None)
+    platform_arn: Optional[str] = field(default=None)
+    template_name: Optional[str] = field(default=None)
+    description: Optional[str] = field(default=None)
+    endpoint_url: Optional[str] = field(default=None)
+    cname: Optional[str] = field(default=None)
+    status: Optional[str] = field(default=None)
+    abortable_operation_in_progress: Optional[bool] = field(default=None)
+    health: Optional[str] = field(default=None)
+    health_status: Optional[str] = field(default=None)
+    resources: Optional[AwsBeanstalkEnvironmentResourcesDescription] = field(default=None)
+    tier: Optional[AwsBeanstalkEnvironmentTier] = field(default=None)
+    environment_links: List[AwsBeanstalkEnvironmentLink] = field(factory=list)
+    operations_role: Optional[str] = field(default=None)
+
+    @classmethod
+    def collect(cls: Type[AwsResource], json: List[Json], builder: GraphBuilder) -> None:
+        def add_tags(env: AwsBeanstalkEnvironment) -> None:
+            tags = builder.client.list(
+                "elasticbeanstalk", "list-tags-for-resource", "ResourceTags", ResourceArn=env.arn
+            )
+            if tags:
+                env.tags = bend(ToDict(), tags)
+
+        def add_resources(env: AwsBeanstalkEnvironment) -> None:
+            resources = builder.client.list(
+                "elasticbeanstalk", "describe-environment-resources", "EnvironmentResources", EnvironmentId=env.id
+            )
+            if resources:
+                env.resources = from_json(
+                    bend(AwsBeanstalkEnvironmentResourcesDescription.mapping, resources),
+                    AwsBeanstalkEnvironmentResourcesDescription,
+                )
+
+        for js in json:
+            instance = cls.from_api(js)
+            builder.add_node(instance, js)
+            builder.submit_work(add_tags, instance)
+            builder.submit_work(add_resources, instance)
+
+    def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
+        super().connect_in_graph(builder, source)
+        builder.dependant_node(
+            self,
+            reverse=True,
+            # delete_same_as_default=True, ??
+            clazz=AwsBeanstalkApplication,
+            name=self.application_name,
+        )
+        res = self.resources
+        if not res:
+            return
+        if res.auto_scaling_groups:
+            for group in res.auto_scaling_groups:
+                if group.auto_scaling_group_name:
+                    builder.dependant_node(
+                        self,
+                        reverse=True,
+                        # delete_same_as_default=True, ??
```

Comment:
do we need this line?

Reply:
I don't know - hence the questionmarks 💡 `delete_same_as_default` is `False` per default. But I don't know if it should be `True` here?

Reply:
I think if it works on the live cloud, then we don't need it, right?